app/views/components/forms/_custom.html.erb

```diff
@@ -127,5 +116,16 @@
         <% end %>
       </div>
     <% end %>
+    <%= hidden_field_tag :authenticity_token, form_authenticity_token if form.verify_csrf? -%>
```

Comment:
looks like these hidden fields were moved.

how were they impacting the tabbing?
is it only ordering?

Reply:
Yes, only the ordering changes. The linked USWDS ticket tries to explain why hidden inputs are affecting the tabbing. We can avoid tabbing issues as long as the hidden inputs are listed somewhere in the middle of the modal code, neither as the first element or the last element.

Reply:
👍